src/Routes.js

```diff
@@ -23,14 +23,14 @@ import { authChange } from 'services/firebase/firebase';
 import {
 	logInUser,
 	signOutUser,
+	toggleDarkMode,
 	toggleLoading
 } from 'store/slice/auth.slice';
 import FormAdmin from 'views/FormAdmin/FormAdmin';
 import Loading from 'views/Loading/Loading';
 import CreateFormPopup from 'views/FormAdmin/CreateFormPopup';
-import { getAllForms } from 'utils/form-data/form-data';
-import { addForms } from 'store/slice/forms.slice';
 import JoinForm from 'views/JoinForm/JoinForm';
+import { getInitialTheme } from 'utils/theme-handler';
 
 const Routers = () => {
 	const auth = useSelector((state) => state.auth);
```

Comment:
Always extract the properties from redux selector otherwise your component will re-render every time something changes in `auth` slice,  no matter you are listening for that property or not.